### DISCUSSION TITLE
Custom emojis

### DISCUSSION THREAD

**Discussion by mplatypus** [squidfunk/mkdocs-material#7186] [Q&A] [answered]
Hi, as the title says, I was wondering if there is any method for adding custom emojis to the site?

```
![image](../assets/image.png){ height="18" width="18"} 
```

I am currently doing the above, but it is not nicely centered on the page. I can convert/get the image I am using in svg if needed aswell.

**Comment by squidfunk** [comment, marked as answer]
You can just use the `twemoji` class, which our emoji integration uses – no need for explicit `width` or `height`:

``` md
![image](../assets/image.png){ .twemoji } 
```

**Reply by mplatypus** [reply]
Thank you for this.
